profiles/source/Source_AudioSettings.yaml

```diff
@@ -73,7 +73,7 @@ dsAudio:
             # dsMS12SUPPORT_GraphicEqualizer           = 0x200, ///< MS12 Graphic equalizer
             # dsMS12SUPPORT_LEConfig                   = 0x400, ///< MS12 LE config
             # dsMS12SUPPORT_Invalid                    = 0x800  ///< Invalid / Out of range
-            MS12_Capabilities: 0x05  # DolbyVolume | IntelligentEqualizer | DialogueEnhancer
+            MS12_Capabilities: 0x4D  # DolbyVolume | IntelligentEqualizer | DialogueEnhancer
```

Comment:
this configuration file should enable everthing.

To run the test with the configuration file specific to the platform, these are internal and are defined by `deviceSettings,yml` which is also an internal file.